src/drivers/pmw3901/pmw3901.cpp

```diff
@@ -341,6 +344,44 @@ PMW3901::init()
 {
 	int ret = PX4_ERROR;
 
+	/* get sensor rotation in yaw */
+	param_t rot = param_find("SENS_FLOW_ROT");
+
+	if (rot != PARAM_INVALID) {
+		int32_t val = 0;
+		param_get(rot, &val);
+
+		_sensor_rotation = (enum Rotation)val;
+	}
+
+	/* get operational limits of the sensor */
+	param_t hmin = param_find("SENS_FLOW_MINHGT");
+
+	if (hmin != PARAM_INVALID) {
+		float val = 0.7;
+		param_get(hmin, &val);
+
+		_sensor_min_range = val;
+	}
+
+	param_t hmax = param_find("SENS_FLOW_MAXHGT");
+
+	if (hmax != PARAM_INVALID) {
+		float val = 3.0;
+		param_get(hmax, &val);
+
+		_sensor_max_range = val;
+	}
+
+	param_t ratemax = param_find("SENS_FLOW_MAXR");
```

Comment:
This is a fixed quantum for this specific sensor - why do we need to load this from a parameter?

Reply:
This was variable for the PX4Flow, depending on the lens used. Which is why it's historically a parameter.

Reply:
@LorenzMeier @mhkabir  I will test the sensor with a value close to the max indicated rate (to give some room for control error and disturbances).

Reply:
@RomanBapst Could you please wrap this up tomorrow? In general, it would be good to not keep PRs hanging open. Thanks!

Reply:
Maybe another reason why this parameter should stay configurable: At the moment the controller will limit the horizontal velocity to comply with the maximum rotation speed of the flow sensor. However, during braking maneuvers you also need to take into account the angular velocity of the vehicle which can be considerable.

Reply:
But for this sensor it is still fixed. I understand the intent, but I want to specifically eliminate magic tuning parameters. There is a max rate param you can use together with the P term of the controller to estimate a likely max rotation and add it to the limit. Plus I’m not sure if a short-term flow loss is a problem on braking.

At any rate, it does not belong into the driver at all.